I'm not sure what the error is here, and @jonjohnsonjr put up https://github.com/wolfi-dev/os/pull/17554 which was a simple rebuild and saw the error also.

```
Run docker run \
  docker run \
    --privileged \
    --security-opt "seccomp=unconfined" \
    --security-opt "apparmor:unconfined" \
    -v "$PWD":"/github/home" \
    -v /home/runner:/home/runner \
    --workdir "/github/home" \
    --entrypoint "bash" \
    -e "HOME=/github/home" \
    -e "GITHUB_ACTIONS=true" \
    -e "CI=true" \
    -e CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=${CLOUDSDK_AUTH_CREDENTIAL_FILE} \
    -e GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS} \
    -e GOOGLE_GHA_CREDS_PATH=${GOOGLE_GHA_CREDS_PATH} \
    -i \
     \
    ghcr.io/wolfi-dev/sdk:latest@sha256:85532dcdd22df15a4610e1f7b213221f35ff6087327f20d637063849a6028c66 <<EOF
  set -e
  # Create a fake linux fs under /tmp/emptyroot to pass to `apk --root`.
  mkdir -p /tmp/emptyroot/etc/apk
  cp -r /etc/apk/* /tmp/emptyroot/etc/apk/
  cat /dev/null > /tmp/emptyroot/etc/apk/world
  
  mkdir -p /tmp/emptyroot/lib/apk/db
  touch /tmp/emptyroot/lib/apk/db/{installed,lock,scripts.tar,triggers}
  
  mkdir -p /tmp/emptyroot/var/cache/apk
  apk update --root /tmp/emptyroot
  
  # Find .apk files and add them to the string
  for f in \$(find packages -name '*.apk'); do
      tar -Oxf \$f .PKGINFO
      apk add --root /tmp/emptyroot --repository "./packages" --allow-untrusted --simulate \$f
  done
  
  EOF
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
/home/runner/work/_temp/7f6238bb-a3d1-402e-98f5-8a43b0b1572f.sh: line 1: apk: command not found
WARNING: opening ./../../packages: No such file or directory
fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
 [https://packages.wolfi.dev/os]
OK: 52683 distinct packages available
# Generated by melange v0.6.11-25-g9a9816a
pkgname = bash-dev
pkgver = 5.2.21-r2
arch = x86_64
size = 377863
origin = bash
pkgdesc = bash development headers
url = 
commit = f843d0b2abd14a7aa9dd22bbf182fffa28a1fd95
license = GPL-3.0-or-later
provides = cmd:bashbug=5.2.21-r2
provides = pc:bash=5.2-release
datahash = d657db333427babc49434c605800f0e908acb60a0d2820ff469feb00a3cc4ec1
WARNING: opening ./../../packages: No such file or directory
ERROR: packages/x86_64/bash-dev-5.2.21-r2.apk: package file format error
Error: Process completed with exit code 99.
```

--- a/bash.yaml
+++ b/bash.yaml
@@ -1,7 +1,7 @@
 package:
   name: bash
   version: 5.2.21
-  epoch: 1
+  epoch: 2
   description: "GNU bourne again shell"
   copyright:
     - license: GPL-3.0-or-later
@@ -41,6 +41,20 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - name: bash-binsh
+    dependencies:
+      provider-priority: 60
+      runtime:
+        - bash
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/bin
+          ln -s bash "${{targets.subpkgdir}}"/bin/sh
+    test:
+      pipeline:
+        - runs: |
+            [ /bin/sh -ef /bin/bash ] || { echo "/bin/sh is not same as /bin/bash"; exit 1; }
+
   - name: "bash-doc"
     description: "bash documentation"
     pipeline:

